models/topics.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection.js");
+
+function retreiveTopics() {
+    const sqlQuery = `SELECT * FROM topics;`;
+  
+    return db.query(sqlQuery).then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "not found" });
+      }
+      return result;
```

Comment:
ideally here we should send back just the data so result.rows rather than the entire database response object.